std/uuid.d

```diff
@@ -1094,7 +1058,7 @@ static assert(is(typeof(rndGen) == Mt19937));
 /**
  * ditto
  */
-UUID randomUUID(RNG)(ref RNG randomGen) if(isUniformRNG!(RNG) &&
+@trusted UUID randomUUID(RNG)(ref RNG randomGen) if(isUniformRNG!(RNG) &&
```

Comment:
Indeed, but the code does contain a trusted "binary trusted copy". Gonna see if I can't fix this.